BitSliceIndexing/bsi.go

```diff
@@ -474,3 +474,27 @@ func (b *BSI) ClearValues(foundSet *roaring.Bitmap) {
 	}
 	wg.Wait()
 }
+
+// NewBSIRetainSet
+func (b *BSI) NewBSIRetainSet(foundSet *roaring.Bitmap) *BSI {
+
+        newBSI := NewBSI(b.MaxValue, b.MinValue)
```

Comment:
Can you run "go fmt" on this code? It is a small thing, and it is optional,  but it does save me the trouble of doing it later.